pkg/utils/exec.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2024 The Fluid Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// allowPathlist of safe commands
+var allowPathlist = map[string]bool{
+	// "helm":    true,
+	"kubectl": true,
```

Comment:
I think we can use `func init()` to find the kubectl and ddc-helm command path, and put them into the white list dynamically